xds/internal/client/client_test.go

```diff
@@ -71,6 +70,16 @@ var (
 		}),
 		protocmp.Transform(),
 	}
+
+	// When comparing NACK UpdateMetadata, we only care if error is nil, but not
+	// the details in error.
+	errPlaceHolder            = fmt.Errorf("error whose details don't matter")
+	cmpOptsIgnoreErrorDetails = cmp.Options{
```

Comment:
Can this be `cmpOptsIgnoreDetails` instead, since it also ignores the timestamp?